static/app/components/prevent/integratedOrgSelector/integratedOrgSelector.tsx

```diff
@@ -1,55 +1,51 @@
-import {Fragment, useCallback, useMemo} from 'react';
+import {useCallback, useMemo} from 'react';
 import styled from '@emotion/styled';
 
 import {LinkButton} from 'sentry/components/core/button/linkButton';
 import type {SelectOption} from 'sentry/components/core/compactSelect';
 import {CompactSelect} from 'sentry/components/core/compactSelect';
-import {Flex} from 'sentry/components/core/layout';
-import {ExternalLink} from 'sentry/components/core/link/link';
+import {Flex, Grid} from 'sentry/components/core/layout';
+import {ExternalLink} from 'sentry/components/core/link';
+import {Text} from 'sentry/components/core/text';
 import DropdownButton from 'sentry/components/dropdownButton';
 import {usePreventContext} from 'sentry/components/prevent/context/preventContext';
 import {integratedOrgIdToName} from 'sentry/components/prevent/utils';
 import {IconAdd, IconBuilding, IconInfo} from 'sentry/icons';
-import {t} from 'sentry/locale';
+import {t, tct} from 'sentry/locale';
 import useOrganization from 'sentry/utils/useOrganization';
 import {useGetActiveIntegratedOrgs} from 'sentry/views/prevent/tests/queries/useGetActiveIntegratedOrgs';
 
 const DEFAULT_ORG_LABEL = 'Select GitHub Org';
 
-function AddIntegratedOrgButton() {
-  return (
-    <LinkButton
-      href="https://github.com/apps/sentry-io/installations/select_target"
-      size="sm"
-      icon={<IconAdd size="sm" />}
-      priority="default"
-      external
-    >
-      {t('GitHub Organization')}
-    </LinkButton>
-  );
-}
-
 function OrgFooterMessage() {
   return (
-    <Fragment>
-      <AddIntegratedOrgButton />
-      <MenuFooterDivider />
-      <Flex justify="start" gap="md">
-        <IconInfo size="sm" style={{margin: '2px 0'}} />
-        <div>
-          <FooterInfoHeading>
-            Access{' '}
-            <ExternalLink href="https://github.com/apps/sentry-io">
-              GitHub Organization
-            </ExternalLink>
-          </FooterInfoHeading>
-          <FooterInfoSubheading>
-            Ensure admins approve the installation
-          </FooterInfoSubheading>
-        </div>
-      </Flex>
-    </Fragment>
+    <Flex gap="sm" direction="column" align="start">
+      <Grid columns="max-content 1fr" gap="sm">
+        {props => (
+          <Text variant="muted" size="sm" {...props}>
+            <IconInfo size="sm" />
+            <div>
+              {tct(
+                'Installing the [githubAppLink:GitHub Application] will require admin approval.',
```

Comment:
I'd say "GitHub App" instead, seems that's how GH calls it too 
<img width="327" height="144" alt="Screenshot 2025-09-25 at 3 12 31 PM" src="https://github.com/user-attachments/assets/06bbad6b-81f8-4019-b6de-38d2fd7cf1b8" />